v8/quotebacks/quotebacks_mdx.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+# Quotebacks extension for Python-Markdown
+
+Use by including `QuotebacksExtension()` in the extensions list.
+
+Converts Markdown of the form:
+
+```markdown
+> QUOTED-CONTENT
+>
+> -- AUTHOR, [LINKTEXT](URL)
+```
+
+to the quotebacks format.
+"""
+
+
+import logging
+import markdown
+import re
+import sys
+import xml.etree.ElementTree as ET
+
+from markdown import Extension
+
+from nikola.plugin_categories import MarkdownExtension
+
+LOGGER = logging.getLogger("quotebacks-mdx")
+
+# If required, add this at the bottom of the page
+QUOTEBACKS_SCRIPT_TAG = """<script note="" src="https://cdn.jsdelivr.net/gh/Blogger-Peer-Review/quotebacks@1/quoteback.js"></script>"""
+
+
+class QuotebacksExtension(MarkdownExtension, Extension):
+    """Python-Markdown extension that wraps QuotebacksProcessor and
+    makes it available to transform HTML docs.
+    """
+
+    def __init__(self, **kwargs):
+        """ Override the init and set self.config according to:
+
+           https://python-markdown.github.io/extensions/api/
+
+        Keys set in self.config can be passed in as kwargs to this
+        extension to override the default values. These are available
+        from self.getConfigs() (as a dict) later.
+        """
+        # self.config["key"] = [
+        #    [],  # default value
+        #    "Sets ...",   # description of parameter
+        # ]
+        self.config = {}
```

Comment:
Is this required? What can users put here? Is this relevant to a Nikola user? If yes, it should be done via `conf.py`.

Reply:
Removed, after many months.